### PR TITLE
Adds train and validation loss log for tensorboard

### PR DESCRIPTION
**Pull request by higepon**
When running train.py, it was a bit hard to see if the loss is decreasing.
By adding the log, we will be able to see the loss for each buckets on tensorboard.

### CODE REVIEW

**Comment by higepon** [comment]
I had wrong email address in the commit. Closing.
